docs/tutorials/customizing-the-output/custom-dockerfile-change-inbuilt-behavior.md

```diff
@@ -15,169 +15,170 @@ In this example, we look at how to make Move2Kube add custom Dockerfile and a cu
   ```console
-      $ mkdir workspace && cd workspace
+  $ mkdir workspace && cd workspace
   ```
 
-2. Lets use the [enterprise-app](https://github.com/konveyor/move2kube-demos/tree/main/samples/enterprise-app) as input for this flow.
+1. Lets use the [enterprise-app](https://github.com/konveyor/move2kube-demos/tree/main/samples/enterprise-app) as input for this flow.
   ```console
-      $ curl https://move2kube.konveyor.io/scripts/download.sh | bash -s -- -d samples/enterprise-app/src -r move2kube-demos
-
-      $ ls src
-      README.md		config-utils		customers	docs			frontend		gateway			orders
+  $ curl https://move2kube.konveyor.io/scripts/download.sh | bash -s -- -d samples/enterprise-app/src -r move2kube-demos
+  $ ls src
+  README.md		config-utils		customers	docs			frontend		gateway			orders
   ```
 
-3. Let's first run Move2Kube **without** any customization. 
-- If we notice the `Dockerfile` generated for the `frontend` app, it uses `registry.access.redhat.com/ubi8/nodejs-12` as base image
-- There are no scripts named `start-nodejs.sh` in the `frontend` service directory
-- The Kubernetes yamls are generated in `myproject/deploy/yamls` directory
+1. Let's first run Move2Kube **without** any customization. 
+  - If we notice the `Dockerfile` generated for the `frontend` app, it uses `registry.access.redhat.com/ubi8/nodejs-12` as base image
+  - There are no scripts named `start-nodejs.sh` in the `frontend` service directory
+  - The Kubernetes yamls are generated in `myproject/deploy/yamls` directory
   ```console
-      $ move2kube transform -s src/ --qa-skip && ls myproject/source/frontend && cat myproject/source/frontend/Dockerfile && ls myproject/deploy && rm -rf myproject
-
-      Dockerfile		README.md		dr-surge.js		manifest.yml		package-lock.json	server.js		stories			test-setup.js		webpack.common.js	webpack.prod.js LICENSE			__mocks__		jest.config.js		nodemon.json		package.json		src			stylePaths.js		tsconfig.json		webpack.dev.js
-
-      FROM registry.access.redhat.com/ubi8/nodejs-12
-      COPY . .
-      RUN npm install
-      RUN npm run build
-      EXPOSE 8080
-      CMD npm run start
-
-      cicd			compose			knative			knative-parameterized	yamls			yamls-parameterized
+  $ move2kube transform -s src/ --qa-skip
+  $ ls myproject/source/frontend
+  Dockerfile		README.md		dr-surge.js		manifest.yml		package-lock.json	server.js		stories			test-setup.js		webpack.common.js	webpack.prod.js LICENSE		__mocks__		jest.config.js		nodemon.json		package.json		src			stylePaths.js		tsconfig.json		webpack.dev.js
+  $ cat myproject/source/frontend/Dockerfile
+  FROM registry.access.redhat.com/ubi8/nodejs-12
+  COPY . .
+  RUN npm install
+  RUN npm run build
+  EXPOSE 8080
+  CMD npm run start
+  $ ls myproject/deploy
+  cicd			compose			knative			knative-parameterized	yamls			yamls-parameterized
+  $ rm -rf myproject
```

Comment:
This was intentionally made into a single command, because otherwise users will end up trying only the first ls command and miss out on the rm. This will make the tutorial fail in the next command.